### DISCUSSION TITLE
How to render a ruby variable alongside other content in a block?

### DISCUSSION THREAD

**Discussion by quaasy** [yippee-fun/phlex#579] [Help] [answered]
Hello,
I want to render something like this:
```html
<ul>
  <li>
    <span class="icon"></span>
    Here the text
  </li>
</ul>
```
My code in Phlex looks like this:
```ruby
ul do
  benefits.each do |benefit|
    li do
      span class: "icon"
      benefit
    end
  end
end
```
but this only renders the icon, not the text inside `benefit`. If I leave away the icon span, then the text is shown. Also if I wrap the text inside a span both are shown. But this feels not right.

So is there a way I can have both a tag and dynamic text?

**Comment by joeldrapper** [comment, marked as answer]
Hey @quaasy, you can use the `plain` method to do this.

```ruby
ul do
  benefits.each do |benefit|
    li do
      span class: "icon"
      plain benefit
    end
  end
end
```

**Reply by quaasy** [reply]
Ah. That's what I'm looking for 😄 
Thank you!

Is it already in the documentation?

**Reply by joeldrapper** [reply]
Yes. https://www.rubydoc.info/gems/phlex/Phlex%2FSGML:plain

I really need to improve the guide, though. The API docs are only really good for reference if you already know what you're looking for.
